catalog-rest-service/src/main/resources/json/schema/operations/pipelines/airflowPipeline.json

```diff
@@ -46,9 +26,113 @@
             "databaseServiceMetadataPipeline",
             "databaseServiceQueryUsagePipeline"
```

Comment:
Do we need the `schema` field here? We are already requesting the `pipelineType` in the properties. Having the necessary `ingestionConfig` seems enough to me 🤔